test.rb

```diff
@@ -0,0 +1,19 @@
+a = 1
```

Comment:
__[staging][rubocop]__
* Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->